scripts/stack/copy-config.sh

```diff
@@ -17,7 +17,7 @@ fi
 
 is_example_config=false
 if [[ "${CONFIG_PATH:-}" == "" ]]; then
-    CONFIG_PATH="$PROJECT_ROOT/config.yml"
+    CONFIG_PATH="$PROJECT_ROOT/config.yml.example"
```

Comment:
I'd rather not do this, because if the user has been running from source, they may have already made the config changes they want to `config.yml`, and it would be nice for them to still pick these up.

Reply:
Make sense.